lib/plugins/ar2.js

```diff
@@ -91,10 +91,10 @@ function init() {
       }
     }
 
-    return ar2.forecast(rawSGVs);
+    return ar2.forecast(rawSGVs, sbx);
   }
 
-  ar2.forecast = function forecast(sgvs) {
+  ar2.forecast = function forecast(sgvs, sbx) {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ['sbx' is defined but never used.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/772273499/issues/source?bid=2452379&fileBranchId=2479882#l97)